### PR TITLE
Advanced Search

### PR DESCRIPTION
**Pull request by James-Crean**
Adds an Advanced Search page. Also adds a way to filter metadata tags since CMIP6 will render scrolling through them ineffective.

### CODE REVIEW

File: static/js/advanced_search.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    form = form.concat("&ajax=true"); 
+    $.ajax({
+        type: 'POST',
+        url: '/advanced_search',
+        data: form,
+        success: function(result){
+            $('#result-count').text(result.count);
+        },
+    }).fail(function($xhr) {
+        console.log('Failed to get count of search results');
```
Comment (mattben) [inline static/js/advanced_search.js]: Should we write errors to a log file? Console is great for debugging but lost at exaction time.

File: static/js/doi.js [hunk 2]
```diff
@@ -36,6 +37,28 @@ $( "#publication-form-wrapper" ).on('change', '#id_pub-status', function() {
...
+    else{
+        //hide tab (and change active tab if needed)
+        $("#meta-tabs div ul li") //too many selected
+    }
+    
+});
+
+var meta = $('.meta-form-list ul li');
+$("#meta-filter").keyup(function() {
+    console.log("hello");
```
Comment (mattben) [inline static/js/doi.js]: Please remove debug statement.

File: static/js/editform.js [hunk 3]
```diff
@@ -30,6 +31,28 @@ $( "#id_status" ).change(function() {
...
+
+$( '#meta-tabs' ).on( "tabsactivate", function( event, ui ) {
+        $(ui.oldTab).removeClass('active');
+        $(ui.newTab).addClass('active');
+        $('#meta_type').val($(ui.newTab).text());
+});
+
+var meta = $('.meta-form-list ul li');
+$("#meta-filter").keyup(function() {
+    console.log("hello");
```
Comment (mattben) [inline static/js/editform.js]: Please remove debug statement.

File: static/js/doi.js [hunk 4]
```diff
@@ -36,6 +37,28 @@ $( "#publication-form-wrapper" ).on('change', '#id_pub-status', function() {
     isDoiRequired();
 });
 
+$( "#publication-form-wrapper" ).on('change', '.project-checkbox', function(e) {
+    var is_checked = $(e.target).is(":checked");
+    if(is_checked){
+        //show tab
```
Comment (mattben) [inline static/js/doi.js]: Maybe change logic around so no to have a empty if statment.

File: static/js/advanced_search.js (hunk 1 above)
Comment (James-Crean) [reply static/js/advanced_search.js]: I am open to suggestions on this. Do you have a method of doing this in mind?

**Pull request merged**